Procfile.sample

```diff
@@ -1,16 +0,0 @@
-## Uncomment lines given your setup and workflow.
```

Comment:
I think this file is referenced in the documentation to quickstart.
It is used by Honcho.

We should remove it in another PR and propose another way to quickstart (without using Honcho and Procfile)